docs/BUILD.md

```diff
@@ -0,0 +1,253 @@
+```
+
+_Warning: Keep the keystore file private; do not check it into public source control._
+
+_Note: `keytool` is part of Java and requires Java to be part of your path_
+
+#### Reference the keystore from the app
+
+Create a file named `key.properties` in the `client/flutter/android` directory that contains a reference to your keystore:
+
+```
+storePassword=<password from previous step>
+keyPassword=<password from previous step>
+keyAlias=key
+storeFile=<location of the key store file, such as /Users/<user name>/key.jks>
+```
+
+_Warning: Keep the key.properties file private; do not check it into public source control._
+
+#### Configure signing in gradle
+
+Configure signing for your app by editing the `build.gradle` file in the `client/flutter/android/app` directory.
+
+1. Add code before android block:
+
+```
+   android {
+      ...
+   }
+```
+
+With the keystore information from your properties file:
+
+```
+   def keystoreProperties = new Properties()
+   def keystorePropertiesFile = rootProject.file('key.properties')
+   if (keystorePropertiesFile.exists()) {
+       keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
+   }
+
+   android {
+         ...
+   }
+```
+
+Load the `key.properties` file into the `keystoreProperties` object.
+
+2. Add code before buildTypes block:
+
+```
+   buildTypes {
+       release {
+           // TODO: Add your own signing config for the release build.
+           // Signing with the debug keys for now,
+           // so `flutter run --release` works.
+           signingConfig signingConfigs.debug
+       }
+   }
+```
+
+With the signing configuration info:
+
+```
+   signingConfigs {
+       release {
+           keyAlias keystoreProperties['keyAlias']
+           keyPassword keystoreProperties['keyPassword']
+           storeFile keystoreProperties['storeFile'] ? file(keystoreProperties['storeFile']) : null
+           storePassword keystoreProperties['storePassword']
+       }
+   }
+   buildTypes {
+       release {
+           signingConfig signingConfigs.release
+       }
+   }
+```
+
+Configure the `signingConfigs` block in your module’s `build.gradle` file.
+
+Release builds of your app will now be signed automatically.
+
+### Reviewing the app manifest
+
+Review the default [App Manifest](https://developer.android.com/guide/topics/manifest/manifest-intro) file, `AndroidManifest.xml`, located in `client/flutter/android/app/src/main` and verify that the values are correct, especially the following:
+
+`application`
+
+Edit the `android:label` in the application tag to reflect the final name of the app.
+
+### Reviewing the build configuration
+
+Review the default [Gradle build file](https://developer.android.com/studio/build/#module-level) file, `build.gradle`, located in `client/flutter/android/app` and verify the values are correct, especially the following values in the `defaultConfig` block:
+
+`applicationId`
+
+Specify the final, unique (Application Id)appid
```

Comment:
They should tell us the app id, we should set it and never touch it again.

Reply:
removed whole section